app/uk/gov/hmrc/economiccrimelevyregistration/controllers/GrsContinueController.scala

```diff
@@ -119,7 +128,8 @@ class GrsContinueController @Inject() (
     bvResult: Option[BusinessVerificationResult],
     grsResult: GrsRegistrationResult,
     entityType: EntityType,
-    mode: Mode
+    mode: Mode,
+    registration: Registration
```

Comment:
you have the request present in the function, you could update it to be a RegistrationDataRequest and use the registration from that

Reply:
Fixed.